business/definitionService.js

```diff
@@ -128,6 +128,7 @@ class DefinitionService {
   }
 
   async _store(definition) {
+    definition.updated = new Date()
```

Comment:
A few questions about this
* should we be adding this to the schema? If the tests are not failing now then the schema is likely wrong. Should not allow additional properties
* In the future we might have more of these sort of `meta` properties. Wanna group them into a parent prop? (e.g., _meta or some such? Suggest we do NOT use _metadata as that will conflict/confuse in some of our store providers)

Reply:
added to schema and grouped in `_meta` object